### DISCUSSION TITLE
How to remove imported rendering file?

### DISCUSSION THREAD

**Discussion by krzysztofjanski** [osmandapp/OsmAnd#12142] [Q&A]
How to remove imported rendering file?
OsmAnd 4.0.5 on iOS, I have imported OpenFietsMap render style http://www.openfietsmap.nl/installation/android/osmand/OpenFietsMap.render.xml?attredirects=0&d=1
now when I select this style OsmAnd is crashing. I do not see any option to remove the style.

**Comment by vshcherb** [comment]
Well, we need to fix the crash, so you can create an issue for that.
1. You can reset Settings to default (iPhone -> Settings -> Apps -> select all default settings) and see if it starts without crashes
If it's still crashes on Start, we need to fix then UI for default settings on iOS.

Short answer to your question: It's not possible to delete rendering style on iOS. Probably it's possible to reimport style with same name and override it.

**Reply by krzysztofjanski** [reply]
Thank you for the answer.
It’s not crashing on start, but when the render style is enabled. As I chose this style for custom profile, the crash happened when the profile was selected. I did remove the profile and imported it again from backup with different render style, and have only to watch out for the list of styles to not tap the bad one (thanks for the w/a with overwrite). I’ll report the issue in some time soon.
